docs/recipes/gulp/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const gulp = require('gulp');
+const connect = require('gulp-connect');
+const lighthouse = require('../../../lighthouse-core');
+const perfConfig = require('../../../lighthouse-core/config/perf.json');
+const port = 8080;
+
+const connectServer = function() {
+  return connect.server({
+    root: '../public',
+    livereload: true,
+    port: port
+  });
+};
+
+function handleOk() {
+  connect.serverClose();
+  process.exit(0);
```

Comment:
I was thinking from the perspective of a user that wants to "fail the build" if their LH score drops to something too low or if they want to use the score in some way. Maybe just add the results object param to `handleOk(results)` and stick a  `// TODO: use lighthouse results` after `connect.serverClose();`?